package-overrides/npm/Leaflet.vector-markers@0.0.6.json

```diff
@@ -1,4 +1,6 @@
 {
-  "dependencies": {},
+  "dependencies": {
+    "leaflet": "^1.0.0"
```

Comment:
Oops, missed that. Leaflet 1.0.x is stable for some time, but maintainer of this package haven't updated its leaflet major version yet. It works fine with it. What is the preferred solution for such situation?